api/v1beta1/ovncontroller_types.go

```diff
@@ -94,6 +94,9 @@ type OVNControllerStatus struct {
 	// NumberReady of the OVNController instances
 	NumberReady int32 `json:"numberReady,omitempty"`
 
+	// OVS-NumberReady of the OVNController instances
+	OVSNumberReady int32 `json:"ovsNumberReady,omitempty"`
```

Comment:
if this is for "OVNController instances" why don't You name it like "OVNControllerNumberReady" or something like that? Or should the comment be different and say about ovs-vswitchd + ovsdb-server PODs instances?

Reply:
I think changing the comment should be enough, copy&paste played a dirty trick here